ecosystem/sep-0024.md

```diff
@@ -540,15 +540,15 @@ Name | Type | Description
 `status` | string | Processing status of deposit/withdrawal.
 `status_eta` | number | (optional) Estimated number of seconds until a status change is expected.
 `more_info_url` | string | A URL the user can visit if they want more information about their transaction's status.
-`amount_in` | string | (optional) Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.
-`amount_out` | string | (optional) Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.
-`amount_fee` | string | (optional) Amount of fee charged by anchor.
-`started_at` | UTC ISO 8601 string | (optional) Start date and time of transaction.
-`completed_at` | UTC ISO 8601 string | (optional) Completion date and time of transaction.
-`stellar_transaction_id` | string | (optional) transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal.
+`amount_in` | string | Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.
+`amount_out` | string | Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.
+`amount_fee` | string | Amount of fee charged by anchor.
+`started_at` | UTC ISO 8601 string | Start date and time of transaction.
+`completed_at` | UTC ISO 8601 string | Completion date and time of transaction.  Return null for in-progress transactions.
```

Comment:
The `Return` feels a little odd here, like it's documentation for a function call. We're describing a piece of data and can just say the value it has in the edge case, without eluding to the operations the data is used in.
```suggestion
`completed_at` | UTC ISO 8601 string | Completion date and time of transaction.  Assigned null for in-progress transactions.
```

Reply:
Could also drop the word `Assigned` and make it `Null for ...`.

Reply:
Thanks yeah, poor wording